controllers/workspace/provision/object.go

```diff
@@ -87,13 +84,12 @@ func SyncObject(object runtime.Object, client client.Client, reqLogger logr.Logg
 		diffOpt = cmp.Options{}
 	}
 	if !cmp.Equal(object, found, diffOpt) {
-		reqLogger.Info("    => Updating "+objType.String(), "namespace", objMeta.GetNamespace(), "name", objMeta.GetName())
+		reqLogger.Info("Updating "+objType.String(), "namespace", objMeta.GetNamespace(), "name", objMeta.GetName())
```

Comment:
it may be the stuff we eventually would like to propagate as events, at least when `debug` mode is enabled.
Don't thanks me for commenting the stuff unrelated to the PR.